src/components/commanderOverview/CommanderDetails.tsx

```diff
@@ -48,10 +49,13 @@ export const CommanderDetails = React.memo(function CommanderDetails() {
     const matches = useSelector((state: AppState) =>
         StatsSelectors.getMatchesByCommanderName(state, commander ? commander.name : "", dateFilter)
     );
+    const matchesFilteredByPlayerCount = filterMatchesByPlayerCount(useSelector((state: AppState) =>
```

Comment:
nit: for the purposes of how we are using it, i think we could rename this to "validMatches" for a shorter (but still descriptive) variable name, and then explain in a comment why these are considered valid matches

Reply:
(i see that you have already used "validMatches" in other places ;) )

Reply:
Explained "validness" in the same comment that explains why we're using two different sets of matches.